joschahenningsen/TUM-Live
# Move timestamp to bottom of thumbnail

### Description

Fixes https://github.com/joschahenningsen/TUM-Live/issues/571 by moving the timestamp to the bottom of the thumbnail where typically the contrast is better due to black bars. Also sorted the CSS so any tooltip styling is at the same location.

### Steps for Testing


Probably best to test this locally. This requires database or code changes that are not suitable for production.

Prerequisites:
- 1 Lecturer/Student/Admin
- 1 VoD
- Hardcode a thumbnail URL and interval here: https://github.com/joschahenningsen/TUM-Live/blob/520b0cd41c04cc4b791551cb982d5d14c8eff707/web/ts/TUMLiveVjs.ts#L61
- The URL can refer to any thumbnail such as https://live.rbg.tum.de/api/stream/20065/thumbs/26524

1. Navigate to a Livestream
2. Hover over the time
3. Observe, that the time is on the bottom of the thumbnail.
4. Remove thumbnail and repeat. It should still look the same.
